nxdl/nxdl_to_hdf5.py

```diff
@@ -1597,44 +1633,27 @@ def symbol_args_to_dict(arg_lst):
     #only search in applications and contributed_definitions subdirectories
     if(class_nm):
         for def_subdir in def_subdirs:
-            class_path = os.path.join(def_dir, def_subdir, class_nm + '.nxdl.xml')
-            if(os.path.exists(class_path)):
+            class_path = pathlib.PurePath(def_dir, def_subdir, class_nm + '.nxdl.xml')
+            if pathlib.Path(class_path).exists():

```

Comment:
```python
    if class_path.exists():
```

Reply:
class_path is a PurePath() which has no  .exists() attribute, hence Path()